config/locales/ar.yml

```diff
@@ -3590,7 +3590,6 @@ ar:
     gateway_error: "عملية الدفع فشلت"
     more: "أكثر"
     new_adjustment: "تعديل جديد"
-    new_order_completed: "طلب جديد مكتمل"
```

Comment:
Thanks for cleaning this up. FYI all locale files except default `en.yml` are managed with an external tool ([Transifex](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n))), so you only need to update `en.yml`. But I think it's fine to remove keys in this case 👍